app/assets/css/custom.css

```diff
@@ -144,4 +144,8 @@ h1 {
 
 a.hmrc-language-select__list-item {
     padding: 0.3125em;
+}
+
+details .arrow {
+    display: none!important;
```

Comment:
ah, `!important` it's always dangerous, but it's inside `.arrow` so should be okay